src/main/java/uk/gov/hmcts/reform/pip/account/management/model/CombinedRoles.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.pip.account.management.model;
+
+import lombok.Data;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.ALL_VERIFIED_THIRD_PARTY_CFT_ROLES;
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.ALL_VERIFIED_THIRD_PARTY_CRIME_ROLES;
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.ALL_VERIFIED_THIRD_PARTY_PRESS_ROLES;
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.GENERAL_THIRD_PARTY;
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.VERIFIED;
+
+@Data
+public class CombinedRoles {
```

Comment:
The name of this class feels a bit random. I think this could be made an utility class, in that case you can create public static methods and use the methods directly without having to instantiate the object every time you use it. May be name the class RoleHelper/RoleUtils, etc and make the class final?
Also since you have this new class, can move all the other constants out from the enum class and put them all in the same place, and create static methods for them for consistency.